### DISCUSSION TITLE
I am attempting to use the basic code (JS) example provided by plotly but it is proving hard to get it to work after adding the code to the text editor in the code option. Am I missing something or does it not allow JS to load from inside?

### DISCUSSION THREAD

**Discussion by ddlidded** [requarks/wiki#3276] [Error / Bug Report]
I am attempting to use the basic code (JS) example provided by plotly but it is proving hard to get it to work after adding the code to the text editor in the code option. Am I missing something or does it not allow JS to load from inside?

I am using the markdown editor and add the following code which works perfectly on a plain old HTML file but once added it does nothing but shows blank space. Any ideas?
I have the following code is used and am running wiki js on the following platform:

CODE:
<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
<div id="tester" style="width:600px;height:250px;"></div>
<script>
	TESTER = document.getElementById('tester');
	Plotly.newPlot( TESTER, [{
	x: [1, 2, 3, 4, 5],
	y: [1, 2, 4, 8, 16] }], {
	margin: { t: 0 } } );
</script>

System:
OS: Ubuntu v18
Docker (CapRover)
Wiki.js version: 2.5.170
Database engine: Postgres

_Originally posted by @ddlidded in https://github.com/Requarks/wiki/discussions/2805_

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)
